.github/workflows/nightly-extended-checks.yml

```
@@ -33,3 +33,15 @@ jobs:
 
       - name: Check external links
         run: npm run check:links -- --current-apis --qiskit-release-notes --external
+
+      - name: Start local Docker preview
+        run: |
+          ./start &
+          sleep 20
+      - name: Check API pages render
+        run: |
+          npm run check-pages-render -- \
+            --qiskit-release-noets \
+            --current-apis \
+            --historical-apis \
+            --translations
```

Aesthetic nit but I recently learnt `>` removes newlines

```suggestion
        run: >
          npm run check-pages-render --
            --qiskit-release-noets
            --current-apis
            --historical-apis
            --translations
```